src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/matchxpath/Example1.java

```diff
@@ -5,19 +5,20 @@
       <property name="query" value="//METHOD_DEF[.//LITERAL_PRIVATE and
               following-sibling::METHOD_DEF[.//LITERAL_PUBLIC]]"/>
       <message key="matchxpath.match"
-              value="Private methods must appear after public methods"/>
+              value="Illegal code structure detected."/>
```

Comment:
Please keep original message value.
If it too long to reference in trailing comment. I am ok to have no message in trailing comments.
It is reasonable exception.

Reply:
Please apply to all others

Reply:
Understood. However, how should I define the expected string for the unit tests, given that it references the MSG_KEY value in message.properties set as **Illegal code structure detected**. In the examples provided, the message key value is different for different examples.
**_Like Referring to Example2 unit test case:_**
<img width="899" alt="Screenshot 2024-11-04 at 3 41 52 PM" src="https://github.com/user-attachments/assets/b6d3089b-bb80-4052-94f8-fcf544cc97ee">
we have the following mismatch:
<img width="1076" alt="Screenshot 2024-11-04 at 3 40 59 PM" src="https://github.com/user-attachments/assets/84ac8789-5259-408d-a594-1726618ce4a8">
."

Reply:
please remove usage getCheckMessage and place there hardcoded value of message.

Reply:
Done.